docs/repositories-configure/using-submodules.md

```diff
@@ -2,39 +2,53 @@
 
 [Git submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodules) allow you to keep a Git repository as a subdirectory within another Git repository. Git submodules are helpful in maintaining a shared configuration file for your team, and then applying it to multiple Git repositories.
 
-## Enabling submodules
+By default, Codacy does normal Git clones that **don't include submodules** to ensure that we only clone necessary repositories. If your organization needs to use submodules, you can request Codacy to enable this feature for you.
 
-By default, Codacy does normal Git clones that don't include submodules to ensure that we only clone necessary repositories. However, if your organization needs to use submodules, please contact us at <mailto:support@codacy.com> asking to enable submodules on Codacy.
+## Prerequisites for using submodules
 
-After we enabled submodules for your organization, do the following:
+1.  Contact us at <mailto:support@codacy.com> asking to enable submodules on Codacy.
 
-1.  **If you're using Codacy Self-hosted**, you must [update the license](../chart/maintenance/license.md).
+1.  **If you're using Codacy Self-hosted**, [update your license](../chart/maintenance/license.md).
 
 1.  If your submodules are:
-
-    -   **Public repositories**, make sure that your Git URL uses the HTTPS protocol
+    -   **Public repositories**, make sure that your Git URL uses the HTTPS protocol.
     -   **Private repositories**, make sure that your Git URL uses the SSH protocol.
 
-1.  If you already have repositories on Codacy where you want to use submodules, navigate to the page **Settings** and click the button **Generate New User Key** for each of these repositories.
+## Enabling submodules on a repository {: id="enable"}
```

Comment:
@stefanvacareanu7 and @mrfyda
While working on the documentation updates for [removing GitHub SSH write and read user permissions](https://codacy.atlassian.net/browse/PLUTO-624), I needed to touch the procedure to [enable submodules](https://docs.codacy.com/v11.0/repositories-configure/using-submodules/#enabling-submodules), so I took the chance to include [this missing step](https://codacy.atlassian.net/browse/DOCS-364).

Note: As we are removing permissions for GitHub, I assumed the missing step now applies to **Bitbucket and GitHub** (not required for GitLab).

Can you please validate if this procedure is now correct? 🙏 
Thanks!

Reply:
yes!
the problem is that if you try to add the key you get an error saying that it is already in use

also happens in GitHub
<img width="263" alt="Screenshot 2023-06-29 at 17 50 41" src="https://github.com/codacy/docs/assets/593860/a650c999-2416-4dbc-86b6-7bb0aaa6f004">